code/game/objects/structures/sensor_tower.dm

```diff
@@ -61,3 +61,118 @@
 			SSminimaps.remove_marker(src)
 			SSminimaps.add_marker(src, z, MINIMAP_FLAG_ALL, "neutral_zone")
 
+/obj/structure/sensor_tower_patrol
+	name = "alpha sensor tower"
+	desc = "A tall tower with a sensor array at the top and a control box at the bottom. Has a lengthy activation process with 3 phases."
+	icon = 'icons/obj/structures/sensor.dmi'
+	icon_state = "sensor_loyalist"
+	obj_flags = NONE
+	density = TRUE
+	layer = BELOW_OBJ_LAYER
+	resistance_flags = RESIST_ALL
+	var/current_timer
+	var/generate_time = 50 SECONDS
+	var/total_segments = 3 // total number of times the activation is required
+	var/activate_time = 5 SECONDS // time to start the activation
+	var/deactivate_time = 10 SECONDS // time to stop the activation proccess
+	var/completed_segments = 0 // what segment we are on, (once this hits total, sensor tower segment is finished)
+	var/id = 1
+	var/activated = FALSE // if all segments are finished
+	var/datum/aura_bearer/current_aura
+	var/aura_strength = 3
+	var/aura_range = 8
+
+/obj/structure/sensor_tower_patrol/Initialize()
+	. = ..()
+	GLOB.zones_to_control += src
+	update_icon()
+
+/obj/structure/sensor_tower_patrol/attack_hand(mob/living/user)
+	if(!ishuman(user))
+		return
+	if(user.do_actions)
+		user.balloon_alert(user, "You are already doing something!")
+		return
+	if(user.faction != FACTION_TERRAGOV)
+		balloon_alert(user, "Only TerraGov can do the objective, defend this!")
+		if(current_timer)
+			if(!do_after(user, deactivate_time, TRUE, src))
+				return
+			current_timer = null
+			balloon_alert(user, "You stop the activation process!")
+			update_icon()
+			QDEL_NULL(current_aura)
+			for(var/mob/living/carbon/human/human AS in GLOB.alive_human_list)
+				human.playsound_local(human, "sound/effects/CIC_order.ogg", 10, 1)
+				human.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:center valign='top'><u>OVERWATCH</u></span><br>" + "[src] activation process has been stopped.", /obj/screen/text/screen_text/command_order)
+		return
+	if(!(SSticker.mode?.flags_round_type & MODE_TWO_HUMAN_FACTIONS))
```

Comment:
Why is this not in the beginning ? 
Do you even need that since it only spawns there